web-browsers/browsers-and-dev-tools.md

```diff
@@ -0,0 +1,220 @@
+
+- Right-click on a page and click on "Inspect" from the menu
+- Use the keyboard shortcut `cmd + opt + i`
+- Using the browser menu
+  - In Mozilla Firefox, we use Tools > Browser Tools > Web Developer Tools
+  - In Google Chrome, we use View > Developer > Developer Tools
+
+Once we do that, the Dev Tools show us details about the current page we're on.
+
+![Dev Tools opened on a web page. The web page is a lesson on the Learn platform.](../assets/web-browsers_browsers-and-dev-tools_dev-tools-intro.png)
+
+### Anatomy of the Dev Tools Window
+
+There are two main sections to navigating the Dev Tools window to focus on right now:
+
+1. The tabs, where each tab contains a different set of tools, all useful in different situations
+1. The content area, which shows the details of whatever tab is selected
+
+![Detailed, cropped view of the Dev Tools on the Inspector tab, showing the tabs and the content](../assets/web-browsers_browsers-and-dev-tools_dev-tools-window.png)
+
+| Tab Name               | Description                                                                         |
+| ---------------------- | ----------------------------------------------------------------------------------- |
+| Inspector or Elements  | Contains details on the rendered HTML elements                                      |
+| Console                | Has a JavaScript console that the current page has access to                        |
+| Sources (Chrome)       | Contains details on static resources (HTML, CSS, and JS) loaded by the current page |
+| Network                | Contains details on HTTP requests and responses that occur, in real-time            |
+| Performance            | Contains tools for recording and testing the performance of rendering               |
+| Memory                 | Contains details on memory usage and allocation, and tracking memory leaks          |
+| Storage or Application | Contains details on things like local storage, cookies, cache, etc.                 |
+
+When developing HTML and CSS, the most relevant tab is the Inspector/Elements tab.
+
+Over time, we should explore our Dev Tools. We should open and use Dev Tools whenever we ask ourselves:
+
+- What does the rendered HTML of this website look like?
+  - What kind of HTML tags exist?
+  - What are their attributes?
+  - How are the HTML elements organized? For any given HTML element, what's nested inside it? What element contains it?
+- What does the rendered styling of this website look like?
+  - What are the CSS rules that are applied to our elements?
+  - What is the position of our HTML element?
+  - Why does our element look a certain way?
+
+These questions will be more relevant to us as we dive deeper into front-end development.
+
+### Inspecting Elements
+
+This tool will become more relevant with HTML development, but we can inspect each element in our web page using Dev Tools.
```

Comment:
'this tool' is unclear since no tool has been mentioned. Flip clauses around to avoid needing the 'but'.
```suggestion
We can inspect each element in our web page using Dev Tools. The Inspect tool will become more relevant as we explore HTML development.
```